### ISSUE TITLE
movewindoworgroup does not move floating like movewindow does

### ISSUE THREAD

**Issue by ymnejmi** [hyprwm/Hyprland#4469]
### Hyprland Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch  at commit v0.34.0  ().
Date: 2024-01-01
Tag:

flags: (if any)
```

</details>
<s>I cant see systeminfo in the valid subcommands and it doesnt work even though im on 0.34.0 supposedly..is this maybe a packaging issue with nix?</s>

### Bug or Regression?

Bug

### Description

movewindow moves floating windows to the edge, movewindoworgroup does not regardless of whether theres a group or not.
I personally would not want it to join a group anyway, just default to the movewindow behaviour, i think its a good idea to have  be consistent in such choices.

Theres probably a reason that its not now that I think about it but just thought such an edge case should be documented as I found no relevant issue.

### How to reproduce

- dispatch movewindoworgroup on a floating window
- it doesnt move

**Comment by fufexan** [comment]
`hyprctl systeminfo` was added after the release.

**Comment by ymnejmi** [comment]
on the subject of nix, is the home manager implementation for hyprland complete? because I cant find a way to set the animations anywhere, other than copying my config in extraConfig
I just took the step to hard switch to nix and would like to do everything the "Nix" way if possible.

I would have liked to join the ds to ask around there but I do not have my account verified with a phone number and I dont plan to anytime soon.

**Comment by fufexan** [comment]
> on the subject of nix, is the home manager implementation for hyprland complete? because I cant find a way to set the animations anywhere, other than copying my config in extraConfig
> I just took the step to hard switch to nix and would like to do everything the "Nix" way if possible.

All the Home Manager module does is convert the nix-style config to hyprlang, so you should be able to add everything you need in there. I have animations set up in [my config](https://github.com/fufexan/dotfiles/blob/839902f67fc1fc0ea4083d5b6caa3debdde44254/home/programs/wayland/hyprland/settings.nix#L49) and they work fine.

**Comment by ymnejmi** [comment]
yeah I just tried to copy the ``bind`` syntax and got it to work, ig thats straightforward enough even for me haha
